bindings-graph/bindings-graph.mjs

```diff
@@ -47,7 +47,7 @@ class CanjsDevtoolsBindingsGraph extends StacheElement {
 
 			selectedObj: String,
 			selectedKey: String,
-			bindingsError: String
+			bindingsError: type.convert(String)
```

Comment:
what other type was being passed for this?

Reply:
It could be `null` or `undefined`

Reply:
Then it should be `type.maybe(String)`

Reply:
I'll push the update of this property with bindings graph fix, thanks!